ledger/ledgercore/statedelta.go

```diff
@@ -113,7 +115,7 @@ type StateDelta struct {
 	// ModStateProofNextRound represents modification on StateProofNextRound field in the block header. If the block contains
 	// a valid state proof transaction, this field will contain the next round for state proof.
 	// otherwise it will be set to 0.
-	ModStateProofNextRound basics.Round
+	ModStateProofNextRound basics.Round `codec:"StateProofNext,omitempty"`
```

Comment:
since the consensus name is `StateProofNext` for API, maybe we rename the field to `StateProofNext` as well?
It will be different from `StateProofNextRound` that is used in block headers (matching name here and there was a confusion point and the trigger for the original renaming)